src/server/routes/teams/get-team.ts

```diff
@@ -0,0 +1,84 @@
+import { Request, IReply } from 'hapi'
+import * as Boom from '../../boom'
+import { TeamModel } from '../../models'
+import { TeamResource, UserResource, HackResource, ChallengeResource } from '../../../resources'
+
+export default async function handler(req: Request, reply: IReply) {
+  const teamId = req.params.teamId
+
+  const team = await TeamModel
+    .findOne({ teamid: teamId }, 'teamid name motto members entries')
+    .populate({
+      path: 'members',
+      select: 'userid name',
+    })
+    .populate({
+      path: 'entries',
+      select: 'hackid name challenges',
+      populate: {
```

Comment:
I'm not a fan of Mongoose. It's only here because it was here when @andrewseward started the project. Still, saves me some time.